test/output/gridChoropleth.svg

```diff
@@ -65,6 +65,6 @@
+  <g transform="translate(24,21.5)"><text dy="0.32em" x="10" y="212">CA</text><text dy="0.32em" x="166" y="314">TX</text><text dy="0.32em" x="374" y="365">FL</text><text dy="0.32em" x="478" y="110">NY</text><text dy="0.32em" x="478" y="161">PA</text><text dy="0.32em" x="322" y="161">IL</text><text dy="0.32em" x="426" y="161">OH</text><text dy="0.32em" x="374" y="314">GA</text><text dy="0.32em" x="426" y="263">NC</text><text dy="0.32em" x="374" y="110">MI</text><text dy="0.32em" x="530" y="161">NJ</text><text dy="0.32em" x="374" y="263">VA</text><text dy="0.32em" x="62" y="110">WA</text><text dy="0.32em" x="114" y="263">AZ</text><text dy="0.32em" x="582" y="59">MA</text><text dy="0.32em" x="322" y="263">TN</text><text dy="0.32em" x="374" y="161">IN</text><text dy="0.32em" x="270" y="212">MO</text><text dy="0.32em" x="478" y="212">MD</text><text dy="0.32em" x="322" y="110">WI</text><text dy="0.32em" x="166" y="212">CO</text><text dy="0.32em" x="270" y="110">MN</text><text dy="0.32em" x="426" y="314">SC</text><text dy="0.32em" x="322" y="314">AL</text><text dy="0.32em" x="218" y="314">LA</text><text dy="0.32em" x="322" y="212">KY</text><text dy="0.32em" x="62" y="161">OR</text><text dy="0.32em" x="218" y="263">OK</text><text dy="0.32em" x="530" y="110">CT</text><text dy="0.32em" x="114" y="212">UT</text><text dy="0.32em" x="270" y="161">IA</text><text dy="0.32em" x="62" y="212">NV</text><text dy="0.32em" x="270" y="263">AR</text><text dy="0.32em" x="270" y="314">MS</text><text dy="0.32em" x="218" y="212">KS</text><text dy="0.32em" x="166" y="263">NM</text><text dy="0.32em" x="218" y="161">NE</text><text dy="0.32em" x="374" y="212">WV</text><text dy="0.32em" x="114" y="161">ID</text><text dy="0.32em" x="10" y="365">HI</text><text dy="0.32em" x="530" y="59">NH</text><text dy="0.32em" x="582" y="8">ME</text><text dy="0.32em" x="114" y="110">MT</text><text dy="0.32em" x="582" y="110">RI</text><text dy="0.32em" x="530" y="212">DE</text><text dy="0.32em" x="218" y="110">SD</text><text dy="0.32em" x="166" y="110">ND</text><text dy="0.32em" x="10" y="8">AK</text><text dy="0.32em" x="426" y="212">DC</text><text dy="0.32em" x="478" y="59">VT</text><text dy="0.32em" x="166" y="161">WY</text></g>
```

Comment:
This looks wrong. In main, the dy: -2 replaced the dy: 0.32em, but in this branch dy: -2 is _in addition to_ the 0.32em from the lineAnchor.